src/cmd/cmd.cpp

```diff
@@ -442,7 +442,7 @@ int main(int argc, char **argv)
     }
 #endif
 
-    account->setUrl(url);
+    account->setUrl(hostUrl.toString().endsWith(QLatin1Char('/')) ? QUrl(hostUrl.toString().chopped(1)) : hostUrl);
```

Comment:
Is that really needed? I mean QUrl should not care if it ends with `/` or not.

Reply:
@FlexW For consistency. When running a GUI version, the URL set to an account is without the traling shash.

Reply:
But does it work without it? If it works without it I would prefer to drop it because it would improve readability. What do you think?

Reply:
@FlexW ok